test/src/main/java/org/apache/accumulo/test/upgrade/UpgradeProgressTrackerIT.java

```diff
@@ -225,7 +224,7 @@ public void testUpdates() throws KeeperException, InterruptedException {
   public void testCompleteUpgrade() throws KeeperException, InterruptedException {
     expectVersion(AccumuloDataVersion.get());
     assertFalse(upgradeNodeExists());
-    progressTracker.startOrContinueUpgrade();
+    progressTracker.continueUpgrade();
     assertTrue(upgradeNodeExists());
```

Comment:
```suggestion
    progressTracker.initialize();
    assertTrue(upgradeNodeExists());
    progressTracker.continueUpgrade();
```

Reply:
Fixed UpgradeProgressTrackerIT in 6c7480c